### DISCUSSION TITLE
Add ability to move the "New Tab" Option in the sidebar to the bottom after all the tabs

### DISCUSSION THREAD

**Discussion by Adish08** [zen-browser/desktop#3769] [Ideas]
Instead of being at the top, a choice should be made accessible to users to determine if it should be at the top of open tabs or at the bottom (for me, personally, bottom makes more sense).

**Comment by niderhoff** [comment]
Its in the settings now

**Comment by 12th-devs** [comment]
![Screenshot 2025-02-07 at 06-04-50 Settings](https://github.com/user-attachments/assets/b168dad0-03f0-4dda-8ab7-d6356c94a959)
